modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -41,3 +41,8 @@ platform:
       - <wildcard_ipv4>
       - <wildcard_ipv6>
 ----
+
+[IMPORTANT]
+====
+You can configure only single networking interface for dual-stack networking.
```

Comment:
This module applies to baremetal too, right? If so, we need to make it clear that this note only applies to VSphere.

Reply:
Thanks for the catch! Didn't realize I missed it.